src/ds/d500/d500-color.cpp

```diff
@@ -138,6 +138,12 @@ namespace librealsense
 
         _ds_color_common->register_color_options();
 
+        color_ep.register_option(RS2_OPTION_POWER_LINE_FREQUENCY,
```

Comment:
As I explained in the ticket.
D555e OHM is same as D400.
Meaning use a global shutter and should have `AUTO` value.

Only D585S should not
So looks like we will need 2 PR's

1 for dev to enable only for D555e + D400 and 1 for other repo

Reply:
OK - missed that line in the requirements - I am on it

Reply:
Done - no need for another PR I beleive

Reply:
Nice hack